voxbox/Rakefile

```diff
@@ -27,5 +43,24 @@ namespace :voxpupuli do
 
       Rake::Task[:lint].invoke
     end
+
+    desc 'Who you gonna call? Ghostbusters!'
+    task :ghostbuster do
```

Comment:
maybe add a test to the pipeline that checks if the rake task exists?

Reply:
some more testing would be nice here :D
will check what i can do about it.